doc/articles/features/windows-storage-pickers.md

```diff
@@ -0,0 +1,228 @@
+}
+else
+{
+    // No file was picked or the dialog was cancelled.
+}
+```
+
+**Notes**: While the `SuggestedStartLocation` has no effect, it must be set for UWP. You must declare at least one `FileTypeFilter` - either an extension or `*` for any file type.
+
+### FileOpenPicker - picking multiple files
+
+``` c#
+var fileOpenPicker = new FileOpenPicker();
+fileOpenPicker.SuggestedStartLocation = PickerLocationId.ComputerFolder;
+fileOpenPicker.FileTypeFilter.Add(".jpg");
+fileOpenPicker.FileTypeFilter.Add(".png");
+var pickedFiles = await fileOpenPicker.PickMultipleFilesAsync();
+if (pickedFiles.Count > 0)
+{
+    // At least one file was picked, you can use them
+    foreach (var file in pickedFiles)
+    {
+        global::System.Diagnostics.Debug(file.Name);   
+    }
+}
+else
+{
+    // No file was picked or the dialog was cancelled.
+}
+```
+
+**Notes**: While the `SuggestedStartLocation` has no effect, it must be set for UWP. You must declare at least one `FileTypeFilter` - either an extension or `*` for any file type.
+
+### FileSavePicker
+
+``` c#
+var fileSavePicker = new FileSavePicker();
+fileSavePicker.SuggestedStartLocation = PickerLocationId.ComputerFolder;
+fileSavePicker.SuggestedFileName = "myfile.txt";
+fileSavePicker.FileTypeChoices.Add("Plain Text", new List<string>() { ".txt", ".text" });
+StorageFile saveFile = await fileSavePicker.PickSaveFileAsync();
+if (saveFile != null)
+{
+    // Save file was picked, you can now write in it
+    await FileIO.WriteTextAsync(saveFile, "Hello, world!");
+}
+else
+{
+    // No file was picked or the dialog was cancelled.
+}
+```
+
+**Notes**: While the `SuggestedStartLocation` has no effect, it must be set for UWP. You must declare at least one item for `FileTypeChoices`. Each has a description and one or more extensions.
+
+## Picker configuration
+
+File pickers have various configuration options that customize the experience (see the <a href="https://docs.microsoft.com/en-us/uwp/api/windows.storage.pickers.fileopenpicker" target="_blank">UWP documentation</a> for full list of properties). Not all options are supported on all target platforms, in which case these are ignored.
+
+To set which file type extensions you want to allow, use the `FileTypeFilter` property on `FileOpenPicker` and `FolderPicker`, and the `FileTypeChoices` property on `FileSavePicker`. Extensions must be in the format ".xyz" (starting with a dot). For `FileOpenPicker` and `FolderPicker` you can also include "*" (star) entry, which represents the fact that any file extension is allowed.
+
+Some systems use `MIME` types to specify the file type. Uno includes a list of common predefined mappings (see list in <a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types" target="_blank">MDN Docs</a>). If a MIME type you require is missing, you can provide by adding it in the `Uno.WinRTFeatureConfiguration.FileTypes.FileTypeToMimeMapping` dictionary:
```

Comment:
Question more about implementation than doc.
Why not use Android mapping?
mimeType = MimeTypeMap.Singleton.GetMimeTypeFromExtension(fileType.Substring(1));

Reply:
The main reason I went with our own mapper is that we already use it in some other API (I think drag and drop) and it allows consistent map across platforms - when an extension is on the list, you know it will be resolved the same way everywhere, and on the other hand if it not on the list, you know you need to provide the override. I realize it the Android implementation could provide more maps out of the box, but could also differ across versions, so that would make things less consistent.

Reply:
But when Android changes mapping, it would use these changed mapping in picker. 
E.g. I want jpg files. Your mapping is "image/jpeg" (same as JPEG). But if Android changes mapping to "image/jpg", than Uno would not show JPG files, only JPEG files.
When I use Android MIME mapping, I'm sure that files I want appears in list.
But when I use Uno mapping, I don't know what result will be. So, in short, I should make "*" as file filter in Picker, and check file extension it in app code.